### DISCUSSION TITLE
Is it fine to enable the NOT condition for a very small ecommerce?

### DISCUSSION THREAD

**Discussion by uroybd** [saleor/saleor#15425] [Q&A]
I can see that the code to support the `NOT` condition in the `where` filters are already in place, and is commented out. The TODO says it requires optimizations.

Is it fine if I enable them for a project where there will only be 20-30 products for now?

**Comment by korycins** [comment]
Hey, it should be ok for 20-30 products. I would also suggest first test it on stage instance.
